src/WorldStateCache.js

```diff
@@ -36,16 +37,24 @@ class WorldStateCache extends EventEmitter {
 
   update() {
     this.updating = this.httpGet().then((data) => {
-      this.lastUpdated = Date.now();
-      delete this.currentData;
-      this.currentData = JSON.parse(data);
-      this.updating = null;
-      this.emit('newData', this.platform, this.currentData);
-      return this.currentData;
+      this.twitter.getData().then((twitterData) => {
```

Comment:
hmmm, either before you merge or after, we should refactor this and the one in the twitter cache to leverage the async/await sugar better.